api/v1/helpers/verbs/utils.js

```diff
@@ -565,7 +565,7 @@ function cleanAndStripNulls(obj) {
 
       // if undefined, parentAbsolutePath needs to be set to empty string,
       // to pass swagger's schema validation
-      if (key === 'parentAbsolutePath' && !o[key]) {
+      if ((key === 'parentAbsolutePath' || key === 'createdBy') && !o[key]) {
```

Comment:
I think we should follow our normal pattern of omitting attributes with null or empty values, unless we have a compelling reason